examples/example_5.yml

```diff
@@ -0,0 +1,47 @@
+language: python
+
+sudo: false
+cache:
+  apt: true
+  directories:
+    - $HOME/.cache/pip
+
+addons:
+  apt:
+    sources:
+      - pov-wkhtmltopdf
+      - chef-stable-trusty
+    packages:
+      - python-lxml
+      - wkhtmltopdf
+      - chefdk
+
+python:
+  - "2.7"
+
+env:
```

Comment:
For travis, If you use matrix the `env` is not take into account.
And I have added the `env` tag in each matrix section, see the following example:
 - https://github.com/OCA/maintainer-quality-tools/blob/master/.travis.yml#L20

t2d should work similar to travis itself and we should ignore `env` root if exists `matrix`.

Reply:
Note: If is so complex this behaviour we should stop this develop because we are migrating all repositories to 3.5